.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
- [ ] @monika-soam nice job in following the project requirement, Kindly add `text.md` file to your `.gitignore` file as required by the project

Reply:
Sure, I will get this done in a few and resubmit.
Thank you

Reply:
@VICTOR-CHUKWU . 
Changes are done, please check and let me know so I can proceed further.